src/utils/navigation.js

```diff
@@ -63,7 +63,6 @@ export const findPage = (url) => {
     if (host === githubHost) {
       const meta = pathToMeta(pathname)
       if (
-          meta.org === page.org &&
```

Comment:
Ah, you are right. There is problem in detecting `defaultOrg`. Don't know how I missed this function :)
I'll try to debug more today